xbmc/GUIInfoManager.cpp

```diff
@@ -7174,7 +7174,7 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
       bReturn = g_application.m_pPlayer->GetPlaySpeed() < 0;
       break;
     case PLAYER_FORWARDING:
-      bReturn = g_application.m_pPlayer->GetPlaySpeed() > 1.5;
+      bReturn = g_application.m_pPlayer->GetPlaySpeed() > IPlayer::MaxPlaySpeed();
```

Comment:
better change a couple of things:

- this feature is referred to as tempo, not playspeed
- min / max are currently 0.8 / 1.5
- the diff of .5 is there in order to deal with rounding errors, would be nice if the code reflects this